portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -38,13 +38,13 @@
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
   private final DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+  private final String entityName = "Comment";
```

Comment:
nit: this can be `static`. When changing to static, make sure to adjust the name according to the [style guide](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)